src/native-common/Network.ts

```diff
@@ -7,7 +7,7 @@
  * Native implementation of network information APIs.
  */
 
-import * as RN from 'react-native';
+import * as NetInfo from '@react-native-community/netinfo';
```

Comment:
@erictraut - thoughts on pulling this out to be a reactxp module instead of inside core (as a breaking change). This will keep ReactXP from requiring module linking and make the module optional (like it now is in React Native)

Reply:
What else would we need to pull out? I'm aware of this and WebView. Is there any other functionality? If that's it, then I'd be in support of breaking out those two components into reactxp extensions. Doing that is going to be quite a bit of work though. And as you said, it will be a significant breaking change. Probably necessitating a version bump to 2.0. Are you volunteering to do it? :)

Reply:
I think all that's needed is to move the code into a new module, it shouldn't be a heavy lift in my opinion.  I agree that a major version bump would be needed
The consumer of the module would react-native link on install or manually link the react-native netinfo module.
I can carve out some time on Friday to tackle this if @sbeca would be open to collaboration on their branch

Reply:
It may be a heavier lift than you expect. We'll need to create new projects for the extensions, update all of the samples (including RXPTest), and update the documentation. If you'd prefer to leave the doc changes to me, I can take care of those.

Reply:
netinfo is only deprecated in RN0.60 if I recall correctly, but geolocation is fully removed: `Full removal of WebView & Geolocation, you now need to rely on the extracted versions` - https://github.com/facebook/react-native/releases/tag/v0.60.0-rc.0

Reply:
I don't mind if this stays in core or is moved out to its own ReactXP module. If we do move it out into its own module, then we should consider doing the same for WebView and AsyncStorage.

And I'm happy to receive any collaboration from others on my branch.

Reply:
Likewise, I may not add code but I'll do my best to pull branches and try things clean following updated instructions to make sure they're clean. I have test rigs for everything but windows, and I'm only missing a couple of the oldest android APIs from the full mobile device version skew covered by react-native

Reply:
Just pushed a full removal of RX.Network to this PR
Creates a new module reactxp-netinfo
Tested in iOS Emulator and Physical Android device